enrolment-ui/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Copyright 2020(c) The Ontario Institute for Cancer Research. All rights reserved.
 
-FROM node AS prebuilder
+FROM node:16 AS prebuilder
```

Comment:
Prevents Docker from using Node 18+, as that breaks compatibility with this project. Node 16 appears to be the latest version of Node that will play nice with this.